BusinessCentral.LinterCop/Design/Rule0044AnalyzeTransferField.cs

```diff
@@ -1,11 +1,11 @@
-﻿using Microsoft.Dynamics.Nav.CodeAnalysis;
+﻿#nullable enable
```

Comment:
Related to the Nullable warnings. I'm wondering if it makes sense to enable this for the whole project? What is you take on this?

https://stackoverflow.com/questions/53633538/how-to-enable-nullable-reference-types-feature-of-c-sharp-8-0-for-the-whole-proj

Reply:
This would be possible, but there are probably going to be a lot of warnings, that should be fixed in the same step. For big legacy projects I recommend to enable it on a file-per-file base as you go, but for this project it might be possible in one go.

Reply:
I'll maybe have a look into it, but indeed for now enable it on a file-per-file base seems indeed the best approach, thanks!